lang/de.js

```diff
@@ -0,0 +1,42 @@
+module.exports = {
+  lang: 'de',
+  words: [
+    {
+      // Please do not delete this one 🙏
+      name: 'spooky',
+      emoji: '761602615326146590',
+    },
+    {
+      name: 'bonbon',
+      emoji: '🍬',
+    },
+    {
+      name: 'geist',
+      emoji: '👻',
+    },
+    {
+      name: 'halloween',
+      emoji: '🎃',
+    },
+    {
+      name: 'abend vor allerheiligen',
```

Comment:
Sorry, at the moment multi-part words don't work (see issue #9)

Reply:
Fixed